ci/unit_tests/dialog_xml2json/main_test.py

```diff
@@ -54,5 +54,5 @@ def test_mainValidActions(self):
         with open(outputJsonPath, 'r') as outputJsonFile:
             outputJson = outputJsonFile.read()
 
-        assert expectedJson == outputJson
+        assert json.loads(expectedJson) == json.loads(outputJson)
```

Comment:
You can use `json.load` directly on file-like object.

Reply:
ok, thanks :)